src/compiler/rust.rs

```diff
@@ -2372,7 +2372,7 @@ impl RlibDepReader {
         trace!("Discovering dependencies of {}", rlib.display());
 
         let mut cmd = process::Command::new(&self.executable);
-        cmd.args(["-Z", "ls"])
+        cmd.args(["-Z", "ls=root"])
```

Comment:
Do we want to care for old rustc ?

Reply:
@sylvestre the change is there since 1.74, so this is broken since. I'm not sure how much we want to care for versions of rustc older than 1.74 here?